reflex/state.py

```diff
@@ -108,10 +109,135 @@
     from reflex.components.component import Component
 
 
-Delta = dict[str, Any]
 var = computed_var
 
 
+@dataclasses.dataclass
+class StateDelta:
+    """A dictionary representing the state delta."""
+
+    data: dict[str, Any] = dataclasses.field(default_factory=dict)
+    reflex_delta_token: str | None = dataclasses.field(default=None)
```

Comment:
seems like `client_token` would be more aligned with the existing code. unless i'm misunderstanding what this field is for